akka-actor/src/main/resources/reference.conf

```diff
@@ -350,6 +350,19 @@ akka {
         fallback = "fork-join-executor"
       }
 
+      affinity-pool-executor {
+        # The number of threads that this executor will maintain in its pool
+        num-threads = 8
```

Comment:
I recommend to use the same min-max structure  as fork-join-executor and thread-pool-executor?